--- a/runtimes/CMakeLists.txt  2023-01-25 16:26:24.000000000 -0800
+++ b/runtimes/CMakeLists.txt  2023-01-25 16:26:26.000000000 -0800
@@ -160,10 +160,10 @@
 option(LLVM_INCLUDE_DOCS "Generate build targets for the runtimes documentation." ON)
 option(LLVM_ENABLE_SPHINX "Use Sphinx to generate the runtimes documentation." OFF)
 
-# Use libtool instead of ar if you are both on an Apple host, and targeting Apple.
-if(CMAKE_HOST_APPLE AND APPLE)
-  include(UseLibtool)
-endif()
+# # Use libtool instead of ar if you are both on an Apple host, and targeting Apple.
+# if(CMAKE_HOST_APPLE AND APPLE)
+#   include(UseLibtool)
+# endif()
 
 # This can be used to detect whether we're in the runtimes build.
 set(LLVM_RUNTIMES_BUILD ON)
